spec/glueby/contract/payment_spec.rb

```diff
@@ -57,14 +57,20 @@
   end
 
   describe '#transfer' do
-    subject { Glueby::Contract::Payment.transfer(sender_wallet: sender_wallet, receiver_wallet: receiver_wallet, amount: amount) }
+    subject { Glueby::Contract::Payment.transfer(sender: sender, receiver: receiver, amount: amount) }
 
-    let(:sender_wallet) { wallet }
-    let(:receiver_wallet) { wallet }
+    let(:sender) { wallet }
+    let(:receiver) { wallet }
     let(:amount) { 200_000 }
 
     it { expect { subject }.not_to raise_error }
 
+    it do
+      allow(rpc).to receive(:sendrawtransaction) do |raw_transaction|
+        expect(raw_transaction).to eq('0100000001c57c5e3dfb5a147710a080cb73628b5df12e7d5172abb8824297f41f04793d5c0000000000ffffffff02400d0300000000001976a91485a683e7863456fb1455e09d288800317445167488acb0acf205000000001976a914d596a47cc210a61949e0af4d5dff1099416331dc88ac00000000')
+      end
```

Comment:
I think that directly verifying the output value will clearly convey what items should be verified in this test.

Reply:
OK, I fix it.